cypress/e2e/marc/marc-authority/edit-authority-record/marc-authority-edit-with-wrong-tag-empty-ldr-and-deleted-field.cy.js

```diff
@@ -0,0 +1,114 @@
+import { Permissions } from '../../../../support/dictionary';
+import DataImport from '../../../../support/fragments/data_import/dataImport';
+import JobProfiles from '../../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../../support/fragments/data_import/logs/logs';
+import TopMenu from '../../../../support/fragments/topMenu';
+import Users from '../../../../support/fragments/users/users';
+import MarcAuthorities from '../../../../support/fragments/marcAuthority/marcAuthorities';
+import MarcAuthority from '../../../../support/fragments/marcAuthority/marcAuthority';
+import QuickMarcEditor from '../../../../support/fragments/quickMarcEditor';
+import getRandomPostfix from '../../../../support/utils/stringTools';
+import { JOB_STATUS_NAMES } from '../../../../support/constants';
+
+describe('MARC -> MARC Authority -> Edit Authority record', () => {
+  const testData = {
+    tag040: '040',
+    tag040NewValue: '0',
+    tag380: '380',
+    tag380RowIndex: 7,
+    ldr: {
+      tag: 'LDR',
+      ldrValue23Symbols: '01918cz\\\\a2200325n\\\\450',
+      ldrValue24Symbols: '01918cz\\\\a2200325n\\\\4500',
+    },
+    searchInput: 'C375167 Beethoven, Ludwig van, 1770-1827. 14 variations sur un thème original',
+    searchOption: 'Keyword',
+    calloutLDRMessage:
+      'Record cannot be saved. The Leader must contain 24 characters, including null spaces.',
+    calloutMessage: 'Record cannot be saved. A MARC tag must contain three characters.',
+  };
+
+  const marcFiles = [
+    {
+      marc: 'marcAuthFileForC375167.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+    },
+  ];
+
+  const createdAuthorityIDs = [];
+
+  before('Create test data', () => {
+    cy.loginAsAdmin({
+      path: TopMenu.dataImportPath,
+      waiter: DataImport.waitLoading,
+    }).then(() => {
+      cy.visit(TopMenu.dataImportPath);
```

Comment:
It would be better if you removed cy.visit on line 46. The user visits this page when logging in (cy.loginAsAdmin({
      path: TopMenu.dataImportPath,
      waiter: DataImport.waitLoading,
    })). And forEach is not needed because the array only has one element